dali/operators/util/normal_distribution_op.cc

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/util/normal_distribution_op.h"
+#include "dali/core/static_switch.h"
+
+namespace dali {
+
+
+DALI_SCHEMA(NormalDistribution)
+                .DocStr(R"code(Creates a tensor that consists of data distributed normally.
+This operator can be ran in 3 modes, which determine the shape of the output tensor:
+1. Providing an input batch to this operator resolves in a batch of output tensors, which have the same shape as the input tensors.
+2. Providing a custom shape as an argument resolves in an output batch, where every tensor has the same (given) shape.
+3. Providing no input arguments resolves in an output batch of scalars, one per every input tensor, distributed normally.)code")
```

Comment:
```suggestion
3. Providing no input arguments resolves in an output batch of scalars, distributed normally.)code")
```
There are no inputs, so 0? :P

Reply:
Nice catch ;)